HelpMyStreetFE/HelpMyStreetFE/Models/Account/UserDetails.cs

```diff
@@ -2,11 +2,44 @@
 {
     public class UserDetails //I expect this class to be depricated when we know how to differentiate users
```

Comment:
You can probably do away with this class now and use the one from the nuget package, will probably need an extension method to add initials though but will be neater than a whole new class